packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -858,6 +858,11 @@ class ImportedCluster extends Resource implements ICluster {
    */
   public readonly hasEc2Capacity: boolean;
 
+  /**
+   * Autoscaling group of the cluster
+   */
+  readonly autoscalingGroup?: autoscaling.IAutoScalingGroup;
```

Comment:
```suggestion
   * The autoscaling group added to the cluster if capacity is associated to the cluster
   */
  public readonly autoscalingGroup?: autoscaling.IAutoScalingGroup;
```
Let's align it with other variables' declarations.